docs/getting-started/olmv1_getting_started.md

```diff
@@ -78,7 +78,7 @@ and on the extension upgrade process [here](../tutorials/upgrade-extension.md).
 
 ```bash
 # Update to v0.11.0
-kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"content-management": {"version": "0.11.0"}}}}'
+kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"catalog": {"version": "0.11.0"}}}}'
```

Comment:
Why are you making this change?
When it change
What is the issue?

Reply:
It works as it is today

```
 $ kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-controller/main/config/samples/olm_v1alpha1_clusterextension.yaml
namespace/argocd created
serviceaccount/argocd-installer created
clusterrolebinding.rbac.authorization.k8s.io/argocd-installer-binding created
clusterrole.rbac.authorization.k8s.io/argocd-installer-clusterrole created
clusterrolebinding.rbac.authorization.k8s.io/argocd-installer-rbac-binding created
clusterrole.rbac.authorization.k8s.io/argocd-installer-rbac-clusterrole created
role.rbac.authorization.k8s.io/argocd-installer-role created
rolebinding.rbac.authorization.k8s.io/argocd-installer-binding created
clusterextension.olm.operatorframework.io/argocd unchanged
```

```
$ kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"content-management": {"version": "0.11.0"}}}}'
Warning: unknown field "spec.source.content-management"
clusterextension.olm.operatorframework.io/argocd patched (no change)

```

Reply:
I see 

```
$ kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"content-management": {"version": "0.11.0"}}}}'
Warning: unknown field "spec.source.content-management"
clusterextension.olm.operatorframework.io/argocd patched (no change)

VS

 $ kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"catalog": {"version": "0.11.0"}}}}'
clusterextension.olm.operatorframework.io/argocd patched
```

Reply:
@camilamacedo86  I think it was a mistake in the original doc: `content-management` is not a field in `ClusterExtension`. This snippet doesn't work